data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +517,1010 @@ mission "Unfettered Jump Drive Trading"
+			`	Before long, you arrive at a small shanty town constructed with metal and other scraps the builders could gather nearby. Even considering what it is, it appears recently constructed and especially ramshackle.`
+			choice
+				`	(Keep walking, ready to reach for my gun.)`
+					goto discreet
+				`	"Is everything alright?"`
+				`	"There is no need to be worried, that cache must be well hidden, right?"`
+			action
+				"unfettered rep: Si-Ra" --
+			`	He makes some kind of sign you do not understand, but it seems pretty clear he wants you to keep quiet. Looking around, you notice the two of you have attracted quite a bit of attention from the impoverished locals. While they all seem to have nothing to lose, the vast majority of them are very young and probably not the hardened criminals you had expected.`
+			label discreet
+			action
+				"unfettered rep: Si-Ra" ++
+			`	After skirting the outskirts of the shanty town, you head further into the mountains, and eventually you spot what appears to be a local in the distance. Though he did not seem to be following you, something seems odd about him, but you cannot pinpoint what.`
+			`	Continuing up the mountain, more and more you get the impression that this vault is no simple, small depot. This is confirmed when you pass several security turrets, and, while it is safe to assume that even more defenses are hidden, you wonder why any would be left in plain sight around a "hidden" depot. Crossing a ridge, you begin to notice that some of your surroundings look strange, as the mountain has only been affected by the elements in some places and not others. Suddenly, the armored Hai stops and raises a hand high, palm open. Recognizing the sign, you stop as well.`
+			`	The Hai reaches for some kind of device in his pocket and enters what you assume to be access codes. By the look on his face, it doesn't seem to be working very well. "Blimey, someone else has accessed the vault. This was not expected," he hisses as his grip tightens on his weapon, and he dons his helmet. "Stay here. I will take care of this."`
+			choice
+				`	"Don't you want one extra gun?"`
+					goto wannafight
+				`	"What about the locals? They don't seem friendly."`
+			`	"Those bilge-sucking scum? They would never dare attack a well-equipped swashbuckler like meself. Furthermore, there shouldn't be any around these parts, so far from those hovels." He steps forward, then glances back at you, unprotected and armed only with a laser that seems like a toy in comparison to his. "But maybe ye should come with me, after all."`
```

Comment:
Wha if you’ve gotten other technology like a Pug staff or the Gegno hand to hand? (cannon)

Reply:
~~da captain always has a pistol~~
idk, but good point

Reply:
I'll add another dialog if ya got a proper gun but it'll be  a bunch of checks and we didnt do that yet for a mission iirc
though you should have expected this anyway
another option is the pistol is one of his pistol which he leans you to blend in better, I think I like that more (he doesnt fully trust you yet)

Reply:
That works better, yes.